docs/tempo/website/traceql/construct-query.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Construct a TraceQL query
+menuTitle: Construct a query
+description: Learn how to construct a TraceQL query
+aliases:
+  - /docs/tempo/latest/traceql/construct-query
+weight: 100
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# Construct a TraceQL query
```

Comment:
oh! ok, I think this is the doc I was expecting in `_index.md`. Personally I think this should be the `_index.md` and rework the above into a "future plans" or something doc.

If I'm a user coming to the traceql docs I feel like this is what I want 99% of the time.

Reply:
Done. :) Well something like that.